tsconfig.json

```diff
@@ -9,6 +9,8 @@
     "noUnusedParameters": false,
     "paths": {
       "@shared/*": ["./src/shared/*"]
-    }
+    },
+    "esModuleInterop": true,
+    "jsx": "react"
```

Comment:
Apparently needed these for my IDE

Reply:
What errors are you getting? 🤔  These values are already[ set as defaults](https://github.com/grafana/tsconfig-grafana/blob/master/base.json), we should not need to override them.